packages/node/src/utils/substrate.ts

```diff
@@ -23,7 +23,9 @@ import {
 } from '@subql/types';
 import { last, merge, range } from 'lodash';
 import { BlockContent } from '../indexer/types';
+import { getLogger } from './logger';
 
+const logger = getLogger('substrate');
```

Comment:
We never used `substrate` this category before, Don't think its good idea adding another.
Can you use the same log category as fetch service?